.github/workflows/linux-eic-shell.yml

```diff
@@ -24,7 +24,8 @@ jobs:
             CMAKE_BUILD_TYPE: Release
           - CC: clang
             CXX: clang++
-            CMAKE_BUILD_TYPE: Release
+            CMAKE_BUILD_TYPE: Debug
+            CXXFLAGS: -fprofile-instr-generate -fcoverage-mapping
```

Comment:
What do we need it for?

Reply:
Since the Release build runs at `-O3` compared to `-O0` for the Debug build, there may be difference in how the code is optimized and what gets ultimately executed in what order (or how things are initialized, reused, elided, etc). We care that the unit tests are correct in the Release build, and while we don't expect that there would be any differences between Debug and Release, there can be some (and they typically point to issues to be fixed). These issues introduced by optimization would likely show up in the Release builds first. They are also expected to be potentially different from the issues that the gcc builds may experience.

Reply:
Good points, of course. However, I believe, we didn't run reconstruction with clang in the first place. Rather hoping to catch crashes with -O3, we would be better off getting things working with ASAN/UBSAN instead #514.

Reply:
Actually, why not.